[doc/samples] Client/Server samples to documentation (#2145)
eclipse-ecal/ecal: doc/rst/getting_started/howto/clientserver/protobuf_math.rst

.. include:: /include.txt

.. _getting_started_howto_clientserver_protobuf_math:

==============
Protobuf: Math
==============

> In C++ we also support the protobuf serialization format for the client/server API.

Reviewer: ```suggestion
In C++ eCAL also supports the protobuf serialization format for the client/server API.
```

> That means that for your request and response you can define an interface as protobuf message in order to make it easier to handle requests.

Reviewer: ```suggestion
That means that for the request and response, an interface can be defined as a protobuf message in order to make it easier to handle requests.
```
